salt/metalk8s/kubeadm/init/certs/apiserver.sls

```diff
@@ -0,0 +1,43 @@
+{%- from "metalk8s/map.jinja" import kube_api with context %}
+{%- from "metalk8s/map.jinja" import networks with context %}
+
+{%- set ca_server = salt['mine.get']('*', 'kubernetes_ca_server').keys() %}
+{%- if ca_server %}
```

Comment:
what happen if the state metalk8s.kubeadm.init.certs.ca is run on several servers ?

Reply:
Yes I know if it's run at the same time, that will advertise on several machine it's the goal of https://github.com/scality/metalk8s/issues/632
I find not better way to do this for the moment

Reply:
Ok, as long as it's a known limitation :) and there is a plan to improve, no big deal !